docker/Dockerfile

```
@@ -0,0 +1,28 @@
+ARG PYTHON_DOCKER_TAG
+
+FROM python:${PYTHON_DOCKER_TAG}
+
+ARG GE_EXTRA_DEPS="spark,sqlalchemy,redshift,s3"
+
+ENV PYTHONIOENCODING utf-8
+ENV LANG C.UTF-8
+ENV HOME /home/great-expectations
+ENV PATH /usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:${HOME}/.local/bin
+# Path where the root of the GE project will be expected
+ENV GE_HOME ${HOME}/great_expectations
+
+LABEL maintainer="great-expectations"
+
```

Here are a few more labels.
```suggestion
LABEL org.opencontainers.image.title="Great Expectations"
LABEL org.opencontainers.image.description="Great Expectations. Always know what to expect from your data."
LABEL org.opencontainers.image.version=${GE_VERSION}
LABEL org.opencontainers.image.created=${CREATED}
LABEL org.opencontainers.image.url="https://github.com/great-expectations/great_expectations"
LABEL org.opencontainers.image.documentation="https://github.com/great-expectations/great_expectations"
LABEL org.opencontainers.image.source="https://github.com/great-expectations/great_expectations"
```